### PR TITLE
Enable stack offloading for all OSs

### PR DESCRIPTION
**Pull request by chfast**
Possible fix for #2770. Controversial as usually.

### CODE REVIEW

File: libethereum/ExtVM.cpp [hunk 1]
```diff
@@ -35,13 +35,20 @@ static unsigned const c_depthLimit = 1024;
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
 static size_t const c_singleExecutionStackSize =
 #ifdef NDEBUG
-	12 * 1024;
+	10 * 1024;
 #else
-	33 * 1024;
+	16 * 1024;
```
Comment (subtly) [inline libethereum/ExtVM.cpp]: What system must they be lowered for?

Comment (chfast) [inline libethereum/ExtVM.cpp]: They don't need to. But they can. I would like to keep it quite low to detect issues as early as possible and to keep offloading point as high as possible.

Comment (gavofyork) [inline libethereum/ExtVM.cpp]: doesn't this needlessly reduce efficiency?

Comment (chfast) [inline libethereum/ExtVM.cpp]: This increases efficiency. We're offloading on higher call depth.

**Pull request merged**